rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/CoordinatorBookKeeper.java

```diff
@@ -119,7 +119,7 @@ protected ClusterManager getClusterManagerInstance(ClusterType clusterType, Conf
           throws CoordinatorInitializationException
   {
     String clusterManagerClassName = CacheConfig.getClusterManagerClass(conf, clusterType);
-    log.info("Initializing cluster manager : " + clusterManagerClassName + " for cluster type " + clusterType);
+    log.debug("Initializing cluster manager : " + clusterManagerClassName + " for cluster type " + clusterType);
```

Comment:
I think it is okay to good to keep this log as **info** as we see it only once when the Bks gets started. And, it sometimes helps in debugging as well.

Reply:
We can enable debug level logs while triaging ... this sort of problems should mostly happen during our testing and changing log levels are easy at that time.